app/services/api.ts

```
@@ -0,0 +1,157 @@
+  assertGuard<HTTPError>(err);
+  return !!err.request && !!err.response;
+};
+
+export const checkIsNetworkError = (err: any) => !err.response && err?.request && !err.request?.status;
+
+export const generateAuthHeaderValue = (token: string) => `Bearer ${token}`;
+
+export const mergeSearchParams = (
+  params: Record<string, string | number | boolean | null>,
+  originalSearchParams: Options['searchParams'] = {},
+  override: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)